wa/commands/runstatus.py

```diff
@@ -0,0 +1,298 @@
+import os
+import logging
+from datetime import datetime, timedelta
+
+import colorama
+
+from wa.framework.configuration.core import Status
+from wa import Command, settings
+from wa.framework.output import RunOutput, discover_wa_outputs
+from wa.framework.exception import NotFoundError
+from wa.utils.terminalsize import get_terminal_size
+from wa.utils.doc import format_column
+from wa.utils.log import COLOR_MAP, RESET_COLOR
+
+class StatusCommand(Command):
+
+    name = 'runstatus'
```

Comment:
Rename to just `status` (the same goes for file name).

Reply:
The problem with using `status` is that we already have an output processor with that name which causes a clash.

Reply:
Ah, right. OK, how about instpect instead? I don't want to limit this command to "run" as we might want to overload it later for other things (e.g. like we do with create). Besides, thinking about it, it should really be a verb in the first place.

Reply:
That makes sense, and seems like a good option. The only other one I could think of was 'report' but happy to go with 'inspect'.

Reply:
Yours is better. Let's go with `report`.